addon/kubeflow-operator/manifest.yaml

```diff
@@ -197,6 +197,32 @@ spec:
         name: manifests
         path: cert-manager/cert-manager-crds
     name: cert-manager-crds
+{{ if .Variables.OIDCProviderURL }}
+  - kustomizeConfig:
+      overlays:
+        - application
+      parameters:
+        - name: namespace
+          value: istio-system
+        - name: userid-header
+          value: kubeflow-userid
+        - name: oidc_provider
+          value: {{ .Variables.OIDCProviderURL }}
+        - name: oidc_auth_url
+          value: {{ .Variables.OIDCProviderURL }}/auth
+        - name: oidc_redirect_uri
+          value: http://{{ .Variables.DashboardURI }}/login/oidc
```

Comment:
What is the default value of this if we don't specify anything? Is it the one from the istio ingress gateway?

Reply:
Defaults are working only with Dex running within the cluster:
```
 - oidc_provider=http://dex.auth.svc.cluster.local:5556/dex
 - oidc_redirect_uri=/login/oidc
 - oidc_auth_url=/dex/auth
```